File: jscomp/test/gpr_1063_test.ml



let bla foo bar baz = foo ## (bar ## baz)

let bla2 foo bar baz = foo##bar##baz

let bla3 foo bar baz = foo ## ( (##) bar baz) 

let bla4 foo x y= foo##(method1 x y (*[@bs]*))

let bla5 foo x y = foo##(method1 x y)

File: jscomp/test/gpr_1063_test.js
'use strict';


function bla(foo, bar, baz) {
  return foo["##"](bar, baz);
}

function bla2(foo, _, _$1) {
  return foo.bar.baz;
}

function bla3(foo, bar, baz) {
  return foo["##"](bar, baz);
}

function bla4(foo, x, y) {
  return foo.method1(x, y);
}

function bla5(foo, x, y) {
  return foo.method1(x, y);
}

exports.bla = bla;
exports.bla2 = bla2;
exports.bla3 = bla3;
exports.bla4 = bla4;
exports.bla5 = bla5;
/* No side effect */
